modules/core/test/resources/META-INF/smithy/traits.smithy

```diff
@@ -176,3 +178,13 @@ enum TestOpenEnum {
 intEnum TestOpenIntEnum {
     ONE = 1
 }
+
+structure TestUrlFormFlattened {
+    @urlFormFlattened
+    test: StringList
```

Comment:
this should be accompanied with a smoke test that ensures that the `test` member carries the `urlFormFlattened` trait